versions/app/src/helpers/helpers.js

```diff
@@ -42,7 +43,11 @@ export const convertTimestamp = (dateObj) => {
 
 
 export const gotoWebLink = (link) => {
-    WebBrowser.openBrowserAsync(link)
+    return WebBrowser.openBrowserAsync(link)
```

Comment:
@armandohimself Was this the issue you were having to get the web view for the FAQs?

Reply:
Yes. I tried passing the link through the `<Custom Button>` as an attribute and the app's navigation did not like that. Decided that instead of spending more time on that and since the FAQ page already had something templated out that I would copy, paste, and style it really quickly.